mods/classic-explorer-treeview.wh.cpp

```diff
@@ -0,0 +1,1040 @@
+            // Resize the parent window to fit the client area
+            SetWindowPos(
+                ctrlNotSink, 0, 0, ctrlNotSinkYOffset, rect.right + ctrlNotSinkXOffset + 1,
+                rect.bottom,
+                SWP_NOZORDER | SWP_NOACTIVATE | SWP_NOSENDCHANGING);
+    
+            // Resize the current window to fit the client area of the parent window
+            GetClientRect(ctrlNotSink, &rect); // Get the client area dimensions of the parent window
+            SetWindowPos(hWnd, 0, 0, 0, rect.right, rect.bottom,
+                         SWP_NOZORDER | SWP_NOACTIVATE);
+    
+            // Find the treeview window within the current window
+            HWND treeview = FindWindowEx(hWnd, NULL, L"SysTreeView32", NULL);
+            GetClientRect(hWnd, &rect); // Get the client area dimensions of the current window
+    
+            // Modify the style of the treeview to remove the client edge
+            LONG lStyle = GetWindowLong(treeview, GWL_EXSTYLE);
+            lStyle &= ~WS_EX_CLIENTEDGE;
+            SetWindowLong(treeview, GWL_EXSTYLE, lStyle);
+    
+            // Calculate and set the position and size of the treeview window
+            int themedBandOffset = GetThemedBandOffset(); // Get themed band offset
+            SetWindowPos(treeview, 0, 2, 24 + themedBandOffset,
+                         rect.right - 4, rect.bottom - 26 - themedBandOffset,
+                         SWP_NOZORDER | SWP_NOSENDCHANGING);
+    
+            // Redraw the parent window, current window, and treeview window
+            InvalidateRect(ctrlNotSink, NULL, true);
+            InvalidateRect(hWnd, NULL, true);
+            InvalidateRect(treeview, NULL, true);
+    
+            resizing = false; // Set resizing flag back to false
+        }
+        return S_OK;
+    }
+    break;
+
+    case WM_DESTROY: // Remove the entry in the global vector
+        g_subclassedNSTCs.erase(std::remove_if(
+                                    g_subclassedNSTCs.begin(), g_subclassedNSTCs.end(),
+                                    [hWnd](NSCTExtra ne)
+        {
+            return ne.hWnd == hWnd;
+        }));
+        return 0;
+        break;
+    }
+
+    return DefSubclassProc(hWnd, uMsg, wParam, lParam);
+}
+
+LRESULT CALLBACK TreeViewSubclassProc(_In_ HWND hWnd,
+                                      _In_ UINT uMsg,
+                                      _In_ WPARAM wParam,
+                                      _In_ LPARAM lParam,
+                                      _In_ DWORD_PTR dwRefData)
+{
+    //Set the cursor to the default arrow, so as to prevent the hand cursor being displayed on item hover
+    if (uMsg == WM_SETCURSOR)
+    {
+        SetCursor(LoadCursor(NULL, IDC_ARROW));
+        return TRUE;
+    }
+
+    return DefSubclassProc(hWnd, uMsg, wParam, lParam);
+}
+
+typedef HWND (*NSCCreateTreeview_t)(void*, HWND);
+NSCCreateTreeview_t NSCCreateTreeviewOriginal;
+HWND NSCCreateTreeviewHook(void* pThis, HWND hWnd)
```

Comment:
Calling conventions are missing, which is likely to cause a crash for 32-bit targets. Please add calling conventions or, if the mod doesn't support 32-bit, limit it to 64-bit, which will also make compilation faster. See also:
https://github.com/ramensoftware/windhawk/wiki/Development-tips#hooks-and-calling-conventions